frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -14,7 +14,7 @@ import { ILookupCodeState } from 'reducers/lookupCodeReducer';
 import { IPropertyQueryParams, IProperty } from '.';
```

Comment:
why do we want to invalidate the cache?

Reply:
We cache to improve performance.  Essentially we don't want to keep asking the API to give us the same thing when the last copy is still good enough.  However when that data has been updated, we need a way to say 'no we really need a fresh copy'.  Otherwise what the client sees is old data and often incorrect.